easybuild/easyconfigs/n/ncdf4/ncdf4-1.16-intel-2017a-R-3.4.0.eb

```diff
@@ -11,7 +11,7 @@ toolchain = {'name': 'intel', 'version': '2017a'}
 
 source_urls = [
     'http://cran.r-project.org/src/contrib/',
-    'http://cran.r-project.org/src/contrib/Archive/$(name)s/',
+    'http://cran.r-project.org/src/contrib/Archive/%(name)s/',
```

Comment:
@zao Please change to `https` (also for `homepage` while you're at it), current PR tests are a bit stricter now...

Reply:
Changed.